services/order.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Optional, Dict
+
+from django.db import transaction
+
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+from services.user import get_user_model
```

Comment:
did u create some custom method ?
u could use build in method 
from django.contrib.auth import get_user_model